### PR TITLE
De-dupe JWT, AuthJS, SIW information

### PR DESCRIPTION
**Pull request by robertjd**
## Description:

* For SIW and Auth JS, removed "ref" pages that duplicated content from GitHub.  Now redirects to the GH page.
* Re-title some pages as "Guides" to distinguish from Quickstarts
* Remove the PHP JWT validation guide, link to the GH README instead (which now has a backlink to the AuthN Guide)
* Remove the PHP Widget guide, because it shows a flow that we no longer want to example (using the access token as a session).  Now redirects to the Quickstart

### Resolves:

* [OKTA-146693](https://oktainc.atlassian.net/browse/OKTA-146693)

### CODE REVIEW

File: _source/_code/javascript/okta_sign-in_widget.md [hunk 1]
```diff
@@ -419,8 +425,10 @@ For other questions, please get in touch with <developers@okta.com>.
...
-You can find more information about the Widget on the [Okta Sign-In Widget Reference Page](/code/javascript/okta_sign-in_widget_ref).
+You can find more information about the Widget on the [Okta Sign-In Widget Reference Page][widget-reference].
 
 ## GitHub Project
 
-If you'd like to see the code that powers the Widget, you can [find it on GitHub](https://github.com/okta/okta-signin-widget).
\ No newline at end of file
+If you'd like to see the code that powers the Widget, you can [find it on GitHub](https://github.com/okta/okta-signin-widget).
+
+[widget-reference]: https://github.com/okta/okta-signin-widget
```
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_sign-in_widget.md]: Minor: Can we make this `https://github.com/okta/okta-signin-widget#okta-sign-in-widget` ?

File: _source/_code/javascript/okta_auth_sdk.md [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
 weight: 10
 excerpt: A JavaScript wrapper for Okta's Authentication APIs.
 redirect_from:
   - "/docs/guides/okta_auth_sdk.html"
 ---
 
-## Overview
+## Okta Auth SDK Guide
 
 The Okta Auth SDK builds on top of our [Authentication API](/docs/api/resources/authn.html) and [OAuth 2.0 API](/docs/api/resources/oidc.html) to enable you to create a fully branded sign-in experience using JavaScript.
```
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Technically, this is the OAuth 2.0 API: https://developer.okta.com/docs/api/resources/oauth2.html

That link is to the OpenID Connect API.

File: _source/_code/javascript/okta_auth_sdk.md [hunk 3]
```diff
@@ -1,31 +1,29 @@
...
-If you'd like to read the Auth SDK's reference page, you will find it here: [Okta Auth SDK Reference Page](/code/javascript/okta_auth_sdk_ref).
+If you'd like to explore the entire Auth SDK, please see the [Okta AuthJS Source &amp; API Reference][authjs-reference].
 
 ## Prerequisites
 
-You will need the following things for this quickstart:
+You will need the following things for this guide:
 
 - An Okta org - If you don't have an existing org, register for [Okta Developer Edition](https://developer.okta.com/signup/).
 - An OpenID Connect Application. Instructions for creating one can be found on [this page](https://help.okta.com/en/prev/Content/Topics/Apps/Apps_App_Integration_Wizard.htm), under the "OpenID Connect Wizard" section.
```
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: This Help link doesn't apply to the new Dev Console. If we're assuming that they're using that then... yea we don't have anything better to link them to, unfortunately.

File: _source/_code/javascript/okta_auth_sdk.md (hunk 2 above)
Comment (robertjd) [reply _source/_code/javascript/okta_auth_sdk.md]: Changed to OIDC

File: _source/_code/javascript/okta_auth_sdk.md (hunk 3 above)
Comment (robertjd) [reply _source/_code/javascript/okta_auth_sdk.md]: Correct, we don't have any help pages (yet) for the dev console workflow

Comment (jmelberg-okta) [reply _source/_code/javascript/okta_auth_sdk.md]: Do we have the bandwidth to create these pages next month? Seems like we could knock out creating **Native**, **SPA**, **Web**, and **Service** how-to's pretty quickly.

File: _source/_code/javascript/okta_auth_sdk.md [hunk 4]
```diff
@@ -58,13 +56,13 @@ To initialize the SDK, create a new instance of the `OktaAuth` object:
 
 ~~~ js
 var authClient = new OktaAuth({
-  url: 'https://{yourOktaDomain}.com', 
+  url: 'https://{yourOktaDomain}.com',
   clientId: '0oab4exampleR4Jbi0h7',
-  redirectUri: 'http://localhost:3333' 
+  redirectUri: 'http://localhost:3333'
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Shouldn't we change this to `8080`?

File: _source/_code/javascript/index.md [hunk 5]
```diff
@@ -22,22 +22,24 @@ New to Okta? Our Quick Start Guide will walk you through adding user authenticat
...
+The Auth SDK builds on top of our Authentication API and OAuth 2.0 API to enable you to create a fully branded sign-in experience using JavaScript.  This SDK is meant to be used in front-end applications.
 
 <ul class='code-list'>
-  <li>
-    <span class='code-icon expression-16'></span> <a href='okta_sign-in_widget_ref.html'>Okta Sign-In Widget Reference</a>
-  </li>
-  <li>
-    <span class='fa fa-github'></span> <a href='https://github.com/okta/okta-signin-widget'>Okta Sign-In Widget Source</a>
-  </li>
+  <li><span class='code-icon launch-16'></span> <a href='okta_auth_sdk.html'>Okta Auth SDK Guide</a></li>
```
Comment (jmelberg-okta) [inline _source/_code/javascript/index.md]: nit: This is the only link ending with `.html`.  You should be able to use `/code/javascript/okta_auth_sdk/`

File: _source/_code/php/jwt-validation.md [hunk 6]
```diff
@@ -1,109 +1,3 @@
 ---
-layout: software
```
Comment (jmelberg-okta) [inline _source/_code/php/jwt-validation.md]: Fun fact: When you google `okta-ios-jwt`, this is the first result.

Comment (jmelberg-okta) [reply _source/_code/php/jwt-validation.md]: Also, is this an interm solution to the bigger problem of not having a great place to document the **JWT** libs?

File: _source/_code/javascript/okta_auth_sdk.md (hunk 3 above)
Comment (jakubvul-okta) [reply _source/_code/javascript/okta_auth_sdk.md]: #1 Create them and put them where?
#2 I wonder if tool-tips in the UI would be enough.

Comment (jmelberg-okta) [reply _source/_code/javascript/okta_auth_sdk.md]: 1. Somewhere on D.O.C would make the most sense. Possibly under the `how-to` section.
2. There are a few too many steps to place inside of a tool-tip. Essentially:
  - Create `X`-type app
  - Setup login-redirect URIs
  - Copy `clientID`, `clientSecret`, etc
  - Assign users/ groups

File: _source/_code/javascript/okta_auth_sdk.md (hunk 4 above)
Comment (robertjd) [reply _source/_code/javascript/okta_auth_sdk.md]: As discussed previously, we're only doing 8080 for sample code.  Guides and quickstarts should still be idiomatic.

File: _source/_code/php/jwt-validation.md (hunk 6 above)
Comment (robertjd) [reply _source/_code/php/jwt-validation.md]: We're working to redirect everyone to the new AuthN guide for JWT validation meta-instructions.  That guide then links out to JWT library READMEs for language-specific examples.  For an example, see https://developer.okta.com/authentication-guide/tokens/validating-access-tokens#okta-libraries-to-help-you-verify-access-tokens

**Pull request merged**